include/ninshiki_cpp/utils/circles.hpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2021 ICHIRO ITS
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+// THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#ifndef NINSHIKI_CPP__UTILS__CIRCLES_HPP_
+#define NINSHIKI_CPP__UTILS__CIRCLES_HPP_
+
+#include <opencv2/opencv.hpp>
+#include <vector>
+
+namespace ninshiki_cpp::utils
+{
+
+class Circles
+{
+private:
+
+    std::vector<cv::Point2f> centers;
+    std::vector<float> radiuses;
```

Comment:
I just realize this, but instead of having a `Circles` class that hold list of centers and radiuses, it's better to have a `Circle` class that only hold a center and a radius and then wrap that `Circle` class into a `std::vector`.